DataStandardization/create_data_to_process_files.py

```diff
@@ -34,23 +52,27 @@ def pop_back(file):
 endpoints = sys.argv[1:]
 
 
-my_list = currentWorkingDir.split('/')
-parent_directory = '/'.join(my_list[:-1])
-output_directory = f'{parent_directory}/Data_To_Process_Files/'
+my_list = path_to_list(currentWorkingDir)
+parent_directory = sep_maker().join(my_list[:-1])
```

Comment:
Maybe I'm not understanding something... Why do you use `sep_maker().join()` instead of `os.path.join()` here?